frontend/components/Layout.js

```diff
@@ -159,7 +159,7 @@ const Layout = ({ children }) => {
               <div className="navbar-end">
                 <div className="navbar-item">
                   <div className="buttons">
-                    <Link href="/random" className="button is-default">
+                    <Link className="button is-default" onClick={()=>window.location.replace("/random")}>
```

Comment:
this is somehow strange, because a href does actually the same thing... maybe it's because this is a next/link ...
Then there are probably some settings for this link, to enforce a reload